checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstyleConstants.java

```diff
@@ -28,6 +28,7 @@
 
     public static final String CHECKER_FILTERS_KEY = "sonar.checkstyle.filters";
     public static final String TREEWALKER_FILTERS_KEY = "sonar.checkstyle.treewalkerfilters";
+    public static final String CHECKER_TAB_WIDTH = "sonar.checkstyle.tabWidth";
```

Comment:
Add "checker" after checkstyle.
It is better to be precise.

Reply:
Sure, I have changed it now.